scripts/run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Copyright (c) 2018
+# Mainflux
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+###
+# Runs all Mainflux microservices (must be previously built and installed).
+#
+# Expects that PostgreSQL and needed messaging DB are alredy running.
+# Also, do not forget to run `cd mqtt && npm install` previously.
```

Comment:
Maybe add this `npm install` to `Makefile`.

Reply:
Good idea, let me try :)